plugin/route53/route53.go

```diff
@@ -99,6 +111,62 @@ func (h *Route53) Run(ctx context.Context) error {
 	return nil
 }
 
+func (h *Route53) updateAliases(ctx context.Context) error {
+	errc := make(chan error)
+	defer close(errc)
+
+	var client = new(dns.Client)
+
+	for zName, z := range h.zones {
+		go func(zName string, z []*zone) {
+			var err error
+			defer func() {
+				errc <- err
+			}()
+			for _, hostedZone := range z {
+				for name, alias := range hostedZone.aliases {
+					var r, req *dns.Msg
```

Comment:
I'm interested in this block on how to properly do the query. as we will need to query the authoritative domain for ALIAS. 

One example is if we have an alias like 

```
alias.example.com ALIAS d21v7mkn0v2jz6.cloudfront.net.
```
The DNS server will preform the lookup for us if we query for the record directly from AWS, but in our case, we need to query an external resolver to get the correct IP addresses for **d21v7mkn0v2jz6.cloudfront.net**

Or we can query the authoritative DNS server directly for the A/AAAA records for **alias.example.com** instead of us trying to resolve the alias address.

Any thought on both options?  @yongtang @dilyevsky